cellengine/resources/attachment.py

```diff
@@ -1,58 +1,49 @@
 from __future__ import annotations
-from dataclasses import dataclass, field
-from dataclasses_json.cfg import config
+
+from attr import define, field
 
 import cellengine as ce
-from cellengine.utils.dataclass_mixin import DataClassMixin, ReadOnly
+from cellengine.utils import converter, readonly
 
 
-@dataclass
-class Attachment(DataClassMixin):
+@define
+class Attachment:
     """A class representing a CellEngine attachment.
     Attachments are non-data files that are stored in an experiment.
     """
 
+    _id: str = field(on_setattr=readonly)
     filename: str
-    _id: str = field(
-        metadata=config(field_name="_id"), default=ReadOnly()
-    )  # type: ignore
-    crc32c: str = field(repr=False, default=ReadOnly())  # type: ignore
-    experiment_id: str = field(repr=False, default=ReadOnly())  # type: ignore
-    md5: str = field(repr=False, default=ReadOnly())  # type: ignore
-    size: int = field(repr=False, default=ReadOnly())  # type: ignore
+    crc32c: str = field(on_setattr=readonly, repr=False)
+    experiment_id: str = field(on_setattr=readonly, repr=False)
+    md5: str = field(on_setattr=readonly, repr=False)
+    size: int = field(on_setattr=readonly, repr=False)
 
-    @classmethod
-    def get(cls, experiment_id: str, _id: str = None, name: str = None) -> Attachment:
```

Comment:
What happened to this method?

Reply:
`<Entity>.get()` has been replaced by `client.get_<entity>`, to minimize the surface area of the toolkit. I'm really happy to revert that change if you want, but I think the latter is easier to understand than the former

Reply:
added back to Attachments; pending for other entities